jirate/jira_cli.py

```diff
@@ -505,6 +513,58 @@ def create_from_template(args):
     return (0, True)
 
 
+
+def _create_from_template(args, template):
+    # Cache for issue createmeta information
+    metadata_by_type = {}
+    _subtask = 'Sub-task'  # To prevent case / typos / etc
+
+    # TODO: consider using Jira's bulk issue creation
+    # TODO: support reading arbitrary fields from the template
+    all_filed = []  # We keep issue keys here because we'll need to refresh anyway
+
+    for issue in template['issues']:
+        reserved_fields = ['subtasks']
+        required_fields = ['summary']
+        if 'project' in issue:
+            pname = issue['project']
+        else:
+            pname = args.project.project_name
+        if pname not in metadata_by_type:
+            metadata_by_type[pname] = {}
+
+        creation_fields = _parse_creation_args(issue, required_fields, reserved_fields)
+
+        # Cache all metadata now (so we can debug subtask creation if needed.
+        issuetype = creation_fields['issuetype']
+        if issuetype not in metadata_by_type:
+            metadata_by_type[pname][issuetype] = _metadata_by_type(args.project, issuetype)
+        if 'subtasks' in issue and issue['subtasks']:
+            metadata_by_type[pname][_subtask] = _metadata_by_type(args.project, _subtask)
+        metadata = metadata_by_type[pname][issuetype]
+
+        filed = {}
+        parent = args.project.create(metadata['fields'], **creation_fields)
+        filed['parent'] = parent.key
+
+        if 'subtasks' in issue and issue['subtasks']:
+            # Set once
+            filed['subtasks'] = []
+            for subtask in issue['subtasks']:
+                metadata = metadata_by_type[pname][_subtask]
+                reserved_fields = ['subtasks', 'issuetype', 'parent']
+                # required_fields are the same
+                start_fields = {'issuetype': _subtask, 'project': pname}
+                start_fields['parent'] = parent.key
```

Comment:
**nit (non-blocking):** we can set the parent as we initialize the dictionary, like:
``start_fields = {'issuetype': _subtask, 'project': pname, 'parent': parent.key}``

Reply:
Let's not block for this for now.